src/bio2bel/sources/rhea.py

```diff
@@ -30,11 +31,37 @@
 CHEBI = 'CHEBI'
 
 
-def _participants(g: rdflib.Graph, reaction_uri: rdflib.term.URIRef) -> Tuple[List[dsl.BaseEntity], List[dsl.BaseEntity]]:
+def get_bel() -> pybel.BELGraph:
+    """Get the Rhea data."""
+    # Parse the RDF file
+    g = BIO2BEL_MODULE.ensure_rdf('rhea', VERSION, url=URL)
+    # Get a list of all the reactions in the database
+    # (the bidirectionalReaction criterion is added to ensure that we only recieve the nondirectional version of a given reaction)
+    rxns = g.query(
+        """
+        SELECT ?reaction ?reactionEquation WHERE {
+            ?reaction rh:equation ?reactionEquation .
+            ?reaction rh:bidirectionalReaction ?bdr
+        }
+        """,
+    )
+    rv = pybel.BELGraph(name='Rhea', version=VERSION)
+    # Loop over reactions, adding reaction nodes to rv as we go
+    # Rather than converting to a set (time-consuming), just let the PyBEL graph handle the occasional duplicate
+    for (reaction_uri, _) in rxns:
+        # Retrieve the reactants and products of the reaction
+        participants = _participants(g, reaction_uri)
+        reactants, products = participants['reactants'], participants['products']
+        # Add a reaction node to the BELGraph
+        rv.add_reaction(reactants, products)
```

Comment:
why bother assigning these values from the dictionary? You could put them right in here like
```suggestion
        rv.add_reaction(participants['reactants'], participants['products'])
```